frontend/user/configProfile/configProfileController.js

```diff
@@ -181,6 +182,17 @@
             window.history.back();
```

Comment:
remove this function, once it is not used anymore

Reply:
actually it is being used at config_profile_mobile.html. I guess it is useful when a user tries to get in the other user's profile page